src/pysentimentanalyzer/sentiment_score_plot.py

```diff
@@ -1,14 +1,23 @@
+# imports
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+from pysentimentanalyzer.helper import *
+
+
 def sentiment_score_plot(df, col):
     """
     Creates a binned histogram showing count of reviews
-    ranging from negative to positive sentiments score.
+    ranging from negative to positive sentiment score.
+
+    Provides the most frequent sentiment score and the corresponding percentage
 
     Parameters
     ----------
     df : Pandas DataFrame
         DataFrame containing the reviews
     col: String
-        Column name in the dataframe that contains the reviews 
+        Name of the column that contains th reviews
```

Comment:
```suggestion
        Name of the column that contains the reviews
```

Reply:
I have made this change and pushed the updated commit